docs/quickstart/nym-connect.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_label: "Nym Connect"
+description: "Getting started with Nym connect"
+hide_title: false
+---
+
+# Nym Connect
+
+Follow this guide to install and run Nym connect.
+
+### Step 1. Download
```

Comment:
These steps need a bit of fleshing out - look at the docs for the wallet and copy some of the formatting there :)